swan-lake/integration/get-started.md

```diff
@@ -10,51 +10,58 @@ intro: Let’s set up a Ballerina development environment and write a simple Bal
 
 ## Install Ballerina
 
-[Download](/downloads/) Ballerina based on the operating system you are using and install it.
+[Download](/downloads/) and install Ballerina based on your operating system.
 
->**Tip:** For more information on installing Ballerina, see [Installation options](/downloads/installation-options/).
+>**Tip:** For more information, see [Installation options](/downloads/installation-options/).
 
 ## Set up the editor
 
 Set up a text editor to write Ballerina code.
 
->**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the Ballerina VS Code extension installed. For detailed information of the functionalities of this extension, go to the <a href="https://wso2.com/ballerina/vscode/docs/" target="_blank">Ballerina VS Code extension documentation</a>.
+>**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the Ballerina VS Code extension installed. For more information about the features of this extension, check <a href="https://wso2.com/ballerina/vscode/docs/" target="_blank">Ballerina VS Code extension documentation</a>.
 
 ## Meet `bal`
 
-`bal` is the Ballerina build tool and package manager. Among other things, `bal` helps you to create, build, test, and run your package. The latest `bal` tool version is available with the latest Ballerina installation. 
+`bal` is the Ballerina build tool and package manager. Among other things, it helps you create, build, test, and run your package. The latest version of the `bal` tool is included with the latest Ballerina installation. 
```

Comment:
```suggestion
`bal` is the Ballerina build tool and package manager. Among other things, it helps you create, build, test, and run your package. The latest version of the `bal` tool is included in the latest Ballerina installation. 
```

Reply:
Since the tool isn't quite a key component with the installation (Ballerina itself is), I think "with" is okay.